src/actions/guides/database/querying-deleting.cr

```diff
@@ -360,15 +360,31 @@ class Guides::Database::QueryingDeleting < GuideAction
     `SELECT AVG(users.age) FROM users`
 
     ```crystal
+    # This will return a Float64 | Nil.
+    # The value will be nil if there are no records.
     UserQuery.new.age.select_average
+
+    # This will return a Float64.
+    # The value will be 0 if there are no records.
+    UserQuery.new.age.select_average!
     ```
 
     `SELECT SUM(users.age) FROM users`
 
     ```crystal
+    # Returns Nil if there are no records
     UserQuery.new.age.select_sum
+
+    # Returns 0 if there are no records
+    UserQuery.new.age.select_sum!
     ```
 
+    > The return type for `select_sum` will be `Int64 | Nil` if the column is an `Int`.
+    > If the column is a `Float` it will be `Float64 | Nil`.
+    >
+    > The return type for `select_sum!` will be `Int64` if the column is an `Int`.
+    > If the column is a `Float` it will be `Float64`.
```

Comment:
I think this could go before the code sample and probably doesn't need `>` since I think it is pretty essential knowledge. Thoughts?

Reply:
TBH if fount the use of `>` a little confusing. I figured the block quotes were just being used to call attention because their styles really stand out. I think it's fine either way. I don't mind moving it to the top.

Reply:
Yeah I think I overused block quotes at first 😆. The rough guideline is if there is information that *might* be helpful or to call out best practices that are tangentially related.

Reply:
Actually, I think this should just be deleted. Your other suggestion for a better comment in the code example covers this anyways.